rviz_default_plugins/CMakeLists.txt

```diff
@@ -63,6 +63,7 @@ find_package(interactive_markers REQUIRED)
 find_package(laser_geometry REQUIRED)
 find_package(map_msgs REQUIRED)
 find_package(nav_msgs REQUIRED)
+find_package(nav2_msgs REQUIRED)
```

Comment:
As you discussed, this could be a problem.  I really don't want to add more dependencies to the core, as it is way too big already.

Stepping back, why can this plugin not live in `nav2_rviz_plugins`?  You already have the dependency available there and you have all of the infrastructure.

Reply:
I'd be more than happy to move this message to nav_msgs, so if you're OK with that then, I'll make that PR.

Reply:
I think we should see it moved to `nav_msgs` or have `nav2_msgs` accepted into `common_interfaces`, which is ever is more suitable (sounds like the former). This step ensures the message type being visualized is broad and generic enough to justify shipping with rviz, and is not specific to the nav2 project specifically.